src/web-app/environments/environment.ts

```diff
@@ -4,11 +4,11 @@
 
 export const environment = {
   production: false,
-  API_URL: 'http://172.21.190.115:8081/',
-  AGM_API_URL: '',
+  API_URL: 'http://localhost:8081/',
   mapbox: {
     accessToken:
-      'pk.eyJ1IjoiYnJhbWJsZXlzIiwiYSI6ImNqam82cmIyZjIzcWozcW0wdjh6enB4Y3kifQ.udssCP8HoCZCKddR4FF8FA'
+      'pk.eyJ1IjoiYnJhbWJsZXlzIiwiYSI6ImNqam82cmIyZjIzcWozcW0wdjh6enB4Y3kifQ.udssCP8HoCZCKddR4FF8FA',
```

Comment:
⚠️ We should avoid to publish our token ! 
If you have time, remove it from here and add the required environment parameters in the `README.md` or `INSTALL.md` file !